android/src/main/java/vn/luongvo/kmm/survey/android/ui/screens/survey/views/MultipleChoices.kt

```diff
@@ -0,0 +1,113 @@
+package vn.luongvo.kmm.survey.android.ui.screens.survey.views
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color.Companion.Transparent
+import androidx.compose.ui.graphics.Color.Companion.White
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.unit.dp
+import vn.luongvo.kmm.survey.android.R
+import vn.luongvo.kmm.survey.android.ui.preview.SurveyDetailParameterProvider
+import vn.luongvo.kmm.survey.android.ui.screens.survey.*
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.dimensions
+import vn.luongvo.kmm.survey.android.ui.theme.AppTheme.typography
+
+@Composable
+fun MultipleChoices(
+    answers: List<AnswerUiModel>,
+    onValueChange: (List<String>) -> Unit,
+    modifier: Modifier = Modifier,
+) {
+    var selectedIndexes by remember { mutableStateOf(emptySet<Int>()) }
+    LazyColumn(modifier = modifier.padding(horizontal = 80.dp)) {
+        items(answers.size) { index ->
+            val isSelected = selectedIndexes.contains(index)
+
+            ChoiceItem(
+                answer = answers[index],
+                isSelected = isSelected,
+                onClick = {
+                    selectedIndexes = if (selectedIndexes.contains(index)) {
+                        selectedIndexes.minus(index)
+                    } else {
+                        selectedIndexes.plus(index)
+                    }
```

Comment:
What do you think we make a fun for this logic so less worry it will grow? Also we can keep the ids of answer instead of index for array safety.
```suggestion
fun didPress(Id) {
                    selectedIds = if (selectedIds.contains(id)) {
                        selectedIds.minus(id)
                    } else {
                        selectedIds.plus(index)
                    }
                                        onValueChange(selectedIds)
                    }
```

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/81/commits/380ecba4ce05401559482bd2c3f293f63b6fab68